adb_graphics/specs.py

```diff
@@ -298,6 +298,16 @@ def rainbow12_colors(self) -> np.ndarray:
                           ([15, 18, 20, 25, 50, 60, 70, 80, 85, 90, 100, 120])
         return np.concatenate((grays, ncar))
 
+    @property
+    def rainbow12_reverse(self) -> np.ndarray:
+
+        ''' Default color map for min helicity '''
+
+        grays = cm.get_cmap('Greys', 2)([0])
+        ncar = cm.get_cmap(self.vspec.get('cmap'), 128) \
+                          ([120, 100, 90, 85, 80, 70, 60, 50, 25, 20, 18, 15])
+        return np.concatenate((ncar, grays))
```

Comment:
One thing that could help here is:

```
def rainbow12_reverse(self) -> np.ndarray:
    return np.flip(rainbow12_colors)
```

Reply:
Having trouble with the flip command, I think because the `rainbow12_colors` array isn't a 1-D set of indices, it's a set of 4-value tuples(?).  So in doesn't just flip the order of colors, it flips the 4 values in each color index:
```
array([[1.        , 1.        , 1.        , 1.        ],
       [0.        , 0.17242769, 1.        , 1.        ],
       [0.        , 0.50600805, 1.        , 1.        ],
       [0.        , 0.72518476, 1.        , 1.        ],
       [0.        , 0.91691389, 1.        , 1.        ],
       [0.44128587, 0.88987998, 0.        , 1.        ],
       [0.72303033, 1.        , 0.2259988 , 1.        ],
       [1.        , 0.92704406, 0.        , 1.        ],
       [1.        , 0.73284702, 0.05330818, 1.        ],
       [1.        , 0.41889133, 0.01952186, 1.        ],
       [1.        , 0.17326918, 0.        , 1.        ],
       [1.        , 0.        , 0.88868695, 1.        ],
       [0.95686887, 0.71717346, 0.96259683, 1.        ]])
```
flips to:
```
array([[1.        , 0.96259683, 0.71717346, 0.95686887],
       [1.        , 0.88868695, 0.        , 1.        ],
       [1.        , 0.        , 0.17326918, 1.        ],
       [1.        , 0.01952186, 0.41889133, 1.        ],
       [1.        , 0.05330818, 0.73284702, 1.        ],
       [1.        , 0.        , 0.92704406, 1.        ],
       [1.        , 0.2259988 , 1.        , 0.72303033],
       [1.        , 0.        , 0.88987998, 0.44128587],
       [1.        , 1.        , 0.91691389, 0.        ],
       [1.        , 1.        , 0.72518476, 0.        ],
       [1.        , 1.        , 0.50600805, 0.        ],
       [1.        , 1.        , 0.17242769, 0.        ],
       [1.        , 1.        , 1.        , 1.        ]])
```

Reply:
nevermind, figured it out